src/mesh/coordinates.cxx

```diff
@@ -698,8 +700,10 @@ const FieldPerp Coordinates::Delp2(const FieldPerp &f) {
   int ncz = localmesh->LocalNz;
 
   // Allocate memory
-  auto ft = Matrix<dcomplex>(localmesh->LocalNx, ncz / 2 + 1);
-  auto delft = Matrix<dcomplex>(localmesh->LocalNx, ncz / 2 + 1);
+  Matrix<dcomplex> delft(localmesh->LocalNx, ncz / 2 + 1);
+  Matrix<dcomplex> ft(localmesh->LocalNx, ncz / 2 + 1);
+  static Laplacian *d2lap = Laplacian::defaultInstance();
```

Comment:
Does this need to be static if we're getting the default instance -- I think it should be the same instance each time.